spotbugs-tests/src/test/java/edu/umd/cs/findbugs/nullness/JSpecifyNullabilityTest.java

```diff
@@ -0,0 +1,42 @@
+package edu.umd.cs.findbugs.nullness;
+
+import edu.umd.cs.findbugs.BugCollection;
+import edu.umd.cs.findbugs.test.SpotBugsExtension;
+import edu.umd.cs.findbugs.test.SpotBugsRunner;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.nio.file.Paths;
+
+import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.emptyIterable;
+
+/**
+ * Check if annotation from https://jspecify.dev/ is detected.
+ *
+ */
+@ExtendWith(SpotBugsExtension.class)
+class JSpecifyNullabilityTest {
+
+    @Test
+    void checkedJSpecifyNullableReturn_isOk(SpotBugsRunner spotbugs) {
+        BugCollection bugCollection = spotbugs.performAnalysis(
+                Paths.get("../spotbugsTestCases/build/classes/java/main/CheckedJSpecifyNullableReturn.class"));
+        assertThat(bugCollection, emptyIterable());
```

Comment:
Can you please assert that it doesn't contain a specific bugtype or bugtypes instead of it does not contain any bug at all? If a new, completely independent bugtype is added later which finds a hit in this testcase, this test would break.